src/AstariaRouter.sol

```diff
@@ -538,10 +541,15 @@ contract AstariaRouter is
         IPublicVault.InvalidStates.EPOCH_TOO_HIGH
       );
     }
-
     if (vaultFee > s.maxStrategistFee) {
       revert IAstariaRouter.InvalidVaultFee();
     }
+    if (epochLength <= s.auctionWindow) {
```

Comment:
this is implied from above where we check:
- `s.minEpochLength <= epochLength`
- and when filing `s.auctionWindow < s.minEpochLength` (need to check this in the constructor as well)